src/widgets/TOCWidget.vue

```diff
@@ -22,13 +22,30 @@
       };
     },
     computed: {
+      endpoint() {
+        return 'https://sv-mini-atlas.herokuapp.com';
+      },
+      rootToc() {
+        return `${this.endpoint}/tocs/demo-root.json`;
+      },
       url() {
-        return 'https://sv-mini-atlas.herokuapp.com/tocs/toc.oaf-1.json';
+        // TODO: Temporary (hack) logic until the TOC API hardens.
+        if (this.$route.name === 'reader') {
+          // TODO: Some kind of version specific TOC lookup logic here.
+          return `${this.endpoint}/tocs/toc.oaf-1.json`;
+        }
+        const tocMap = {
+          1: `${this.endpoint}/tocs/toc.oaf-1.json`,
+          2: `${this.endpoint}/tocs/toc.crito-stephanus-jkt-1`,
```

Comment:
needs to be `${this.endpoint}/tocs/toc.crito-stephanus-jkt-1`

Reply:
yeah i've been having a small issue with this (even despite the missing `json` here which i've sorted out), it still 404s for me with this link which I thought must have been it... 

https://sv-mini-atlas.herokuapp.com/tocs/toc.crito-stephanus-jkt-1.json

Could you possibly drop the actual URL in for me as a sanity check?

Reply:
@jhrr Looks like things were a bit inconsistent in how I was naming the files; a few things that I hope will help:

- Let's update the endpoint hostname (https://mini-stack-a-feature-se-j47yu0.herokuapp), deploys commits pushed to the branch powering https://github.com/scaife-viewer/sv-mini-atlas/pull/6
- The new endpoint has a new route (https://mini-stack-a-feature-se-j47yu0.herokuapp.com/tocs/) that lists the URLs for TOCs
- https://mini-stack-a-feature-se-j47yu0.herokuapp.com/tocs/toc.crito-stephanus-jkt-1.json is the correct URL

Still not our "full" API, but hopefully less of a shot in the dark (and we can use `ls` rather than risk me mis-translating filenames 😉 )

Reply:
Sorted, thanks for that.